frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/graphtable/DiscoverAction.java

```diff
@@ -56,7 +56,8 @@ public void actionPerformed(AnActionEvent anActionEvent) {
                 GitVfsUtils.getMacheteFilePath(selectedRepository), project, getGraphTable(anActionEvent),
                 getBranchLayoutWriter(anActionEvent)),
             /* okButtonText */ getString("action.GitMachete.DiscoverAction.discovered-branch-tree-dialog.save-button-text"),
-            /* cancelButtonVisible */ true).show(), NON_MODAL));
+            /* cancelButtonVisible */ true,
```

Comment:
Btw pls bump version to `0.7.1` in version.gradle (to make better use of `machete-status-branch` hook)